### PR TITLE
New login

### PR DESCRIPTION
**Pull request by orcunsagirsoy**
Refactor Login Pages

### CODE REVIEW

File: src/components/LoginPage/GoogleLogin/index.css [hunk 1]
```diff
@@ -7,8 +7,15 @@
   border: none;
   height: 3em;
   margin-top: 0.5em;
+  width: 17em;
```
Comment (Ammar-64) [inline src/components/LoginPage/GoogleLogin/index.css]: Is this responsive

**Review by Ammar-64 — APPROVED** [review]
LGTM

File: src/App.jsx [hunk 2]
```diff
@@ -26,10 +26,14 @@ function App() {
           <Route path="/signup" component={Signup} />
           <Route path="/login" component={LoginContainer} />
           <Route exact path="/donate" component={NewPostForm} />
+          {/* <Route path="/posts" component={Posts} />
```
Comment (Ammar-64) [inline src/App.jsx]: These three lines should be deleted

File: src/App.jsx [hunk 3]
```diff
@@ -26,10 +26,14 @@ function App() {
...
+      <Route path="/donate" component={Donate} />
+      <Route path="/contact" component={Contactus} /> */}
           <Route exact path="/contact" component={Contact} />
           <Route path="/donatePosts" component={PostPageVolunteer} />
           <Route path="/receivePosts" component={PostPageReceiver} />
           <Route exact path="/posts/:id" component={SinglePostPage} />
+          <Route exact path="/application" />
```
Comment (Ammar-64) [inline src/App.jsx:42]: This should be deleted

File: src/components/LoginPage/Login/index.css [hunk 4]
```diff
@@ -5,8 +5,9 @@
   align-items: center;
   border-radius: 0.5em;
   height: 3em;
-  width: 10.88em;
+  width: 17em;
```
Comment (Ammar-64) [inline src/components/LoginPage/Login/index.css]: This one too

**Review by Ammar-64 — CHANGES_REQUESTED** [review]

File: src/components/LoginPage/Login/index.css (hunk 4 above)
Comment (orcunsagirsoy) [reply src/components/LoginPage/Login/index.css]: PTAL

File: src/App.jsx (hunk 3 above)
Comment (orcunsagirsoy) [reply src/App.jsx:42]: PTAL

**Review by Ammar-64 — APPROVED** [review]
LGTM

**Review by betulerkoc — APPROVED** [review]

**Review by Ammar-64 — APPROVED** [review]

**Pull request merged**